GitExtensionsVSIX/GitExtensionsPackage.vsct

```diff
@@ -104,7 +104,7 @@
         <Icon guid="guidImages" id="clone"/>
         <Strings>
           <CommandName>GitExt.Clone</CommandName>
-          <ButtonText>Clone &amp;repository</ButtonText>
+          <ButtonText>Clone repositor&amp;y</ButtonText>
         </Strings>
```

Comment:
There is already binding for `y` in `Create new repositor&amp;y` is it intended?